index.html

```diff
@@ -16,10 +16,133 @@
   <section id="home-screen">
     <h1>ConnectR</h1>
     <img class="logo" src="/img/connect-logo.svg" alt="brand logo" />
-    <img class= "exchange-icon" src="/img/exchange.svg" alt="two arrows representing exchange contact info" />
-    <img class= "profile-placeholder-icon" src="/img/profile-image-placeholder.svg" alt="profile image placeholder" />
-    <img class= "add-contact-icon" src="/img/add-contact.svg" alt="plus sign representing adding a new contact" />
-    <img class= "contact-list-icon" src="/img/add-contact.svg" alt="book representing contact list" />
+    <section id="home-screen-icon-layout">
+      <article class="home-screen-icons">
+        <img class= "icon" src="/img/exchange.svg" alt="two arrows representing exchange contact info" />
+        <p>Exchange Info</p>
+      </article>
```

Comment:
I used 4 different articles here in order to make it responsive with flexbox since each icon has a p tag associated with it, so that's how I grouped them together.